.travis.yml

```diff
@@ -18,7 +18,6 @@ rvm:
   - 2.0.0
   - jruby-19mode
 env:
-  - PUPPET_VERSION="~> 2.7.0"
```

Comment:
I think we should test against `~> 3.0` and `~> 4.0`, seems like a good time to make the change if others agree.

It might be worth pairing the RVM/ENV as well because we don't need to test 1.8.7/1.9.3/jruby-19mode with 4.x, for instance.